src/themes/buble.styl

```diff
@@ -33,7 +33,7 @@ $sidebar-width = 16rem
 
 .markdown-section h1, .markdown-section h2, .markdown-section h3, .markdown-section h4, .markdown-section strong
   color #333
-  font-weight 400
+  font-weight 600
```

Comment:
The default of these is `bold` in the browser. I am pretty sure this is an intended code.